src/components/product-menu/product-menu.js

```diff
@@ -4,46 +4,24 @@ import Tag from '../tag/tag';
 import classnames from 'classnames';
 
 class ProductMenu extends React.PureComponent {
-  buildTag = (item) => {
-    const tagProps = {
-      theme: item.tag,
-      customLabel: item.customTagProps
-        ? item.customTagProps.customLabel
-        : undefined,
-      customTooltipText: item.customTagProps
-        ? item.customTagProps.customTooltipText
-        : undefined,
-
-      customBackground: item.customTagProps
-        ? item.customTagProps.customBackground
-        : undefined,
-      customColor: item.customTagProps
-        ? item.customTagProps.customColor
-        : undefined,
-      customBorder: item.customTagProps
-        ? item.customTagProps.customBorder
-        : undefined
-    };
-    return (
-      <div className="ml-neg3">
-        <Tag {...tagProps} small={true} />
-      </div>
-    );
-  };
-
   render() {
-    const { props } = this;
+    const { tag, customTagProps, lightText, productName, homePage } =
+      this.props;
     return (
       <div className="dr-ui--product-menu">
-        {props.tag && this.buildTag(props)}
+        {tag && (
+          <div className="ml-neg3">
+            <Tag theme={tag} {...customTagProps} small={true} />
+          </div>
+        )}
         <a
-          href={props.homePage}
+          href={homePage}
           className={classnames('txt-fancy txt-l block', {
-            'color-white color-gray-light-on-hover': props.lightText,
-            'color-blue-on-hover': !props.lightText
+            'color-white color-gray-light-on-hover': lightText,
+            'color-blue-on-hover color-text': !lightText
```

Comment:
I'm interested in trying out `color-text` here! I think we can look back at this when we work on PageLayout to see if this is an improvement or distraction. This the only real change to this component (as shown in the updated snapshots), everything else is clean up.